### PR TITLE
Fixed the program controller state transition.

### PR DESCRIPTION
**Pull request by sagarkapare**
Jira - https://issues.cask.co/browse/CDAP-1898

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractTwillProgramController.java [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
   protected final String programName;
   protected final TwillController twillController;
+  private volatile boolean stopRequested = false;
```
Comment (chtyim) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractTwillProgramController.java]: No need to init to `false`.

**Comment by chtyim** [comment]
Just one comment,.

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractTwillProgramController.java (hunk 1 above)
Comment (sagarkapare) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractTwillProgramController.java]: Fixed.

**Comment by sagarkapare** [comment]
Build is running here - http://builds.cask.co/browse/CDAP-RBT206-2

**Comment by sagarkapare** [comment]
Build passed. Merging.

**Pull request merged**